users/forms.py

```
@@ -49,14 +53,35 @@ class MultipleSelect(forms.CheckboxSelectMultiple):
+                
+                # Get country from form
+                country = self.cleaned_data.get("location_country")
+                
+                # Trigger async task to subscribe user
+                subscribe_user.delay(
+                    email=user.email,
+                    name=user.name,
+                    country=country
+                )
+                logger.info(f"Newsletter subscription task queued for {user.email}")
+            except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)